packages/app/src/settings/device/NotificationSettings.vue

```diff
@@ -6,80 +6,114 @@
     <template #description>
       {{ t('settingsPage.notifications.description') }}
     </template>
-    <div class="divide-y border rounded divide-gray-50 border-gray-100 px-[16px]">
+    <div class="border rounded border-gray-100">
       <div
-        v-for="({id, title}) in switches"
-        :key="id"
-        class="py-[16px]"
+        v-if="!props.gql.localSettings.preferences.desktopNotificationsEnabled"
+        class="min-h-[56px] bg-indigo-50 px-[16px] py-[10px] flex flex-wrap justify-between items-center gap-[5px]"
+        data-cy="enable-notifications"
       >
```

Comment:
The top left and right corners should be rounded or the outer panel should have `overflow:hidden` to prevent the background color from overflowing on to the rounded corners.

Also, the Figma shows a bottom border and it shows the border of this box being a tinted indigo.  Not sure it needs the tint since it is part of the larger panel, but might want to check with design.

Reply:
I don't see the overflow issue. 

I tried to fix the border color, but it seems more work than is worth investing (right now). Could be a nice bonus once we knock out the rest of the notification tickets, let's see if we can revisit this when we get there. I think it's pretty minor and unlikely to be a blocker.